app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Any, Callable
+
+
+def cache(func: Callable) -> Any:
```

Comment:
`cache` will rather return the `inner` function, which is `Callable`. You could be more specific here.